tests/tsconfig.json

```diff
@@ -3,6 +3,7 @@
   "include": [
     "components",
     "hooks",
-    "ssr"
+    "ssr",
+    "./jest-setup.ts",
```

Comment:
which file is this?

Reply:
may have been mistakenly committed. removing.